### PR TITLE
[Chore]: Added the travis.yml for CI

### PR DESCRIPTION
**Pull request by adityad85**

### CODE REVIEW

**Review by ArfatSalman — CHANGES_REQUESTED** [review]
Why the CI is failing?

Add a test file and `test` script in your `package.json`.

**Review by ArfatSalman — CHANGES_REQUESTED** [review]
You also need to set the `test` script in the `package.json`.


Also, 

```js
import React from 'react';
import Enzyme, {shallow} from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({adapter: new Adapter()});

import CheckboxWithLabel from '../CheckboxWithLabel';

it('CheckboxWithLabel changes the text after click', () => {
  // Render a checkbox with label in the document
  const checkbox = shallow(
    <CheckboxWithLabel labelOn="On" labelOff="Off" />
);

  expect(checkbox.text()).toEqual('Off');

  checkbox.find('input').simulate('change');

  expect(checkbox.text()).toEqual('On');
});
```

Use an actual test case. Your `App.js` is already exporting a component. Test it.

File: package.json [hunk 1]
```diff
@@ -6,8 +6,15 @@
...
   "license": "MIT",
   "scripts": {
+    "test": "jest",
+    "test:coverage": "jest --coverage",
+    "precommit": "lint-staged",
     "build": "webpack --config ./webpack.prod.config.js --progress --colors",
     "start": "npm run build && NODE_ENV=production node server.js"
+    
+  },
+  "jest": {
```
Comment (anshurathee) [inline package.json]: Have a separate jest.config file in the root of the project and place all the configurations in that file.
http://jestjs.io/docs/en/configuration

File: package.json [hunk 2]
```diff
@@ -6,29 +6,52 @@
...
   "license": "MIT",
   "scripts": {
+    "test": "jest",
+    "test:coverage": "jest --coverage",
+    "precommit": "lint-staged && npm test",
+    "prepush": "npm test",
     "build": "webpack --config ./webpack.prod.config.js --progress --colors",
     "start": "npm run build && NODE_ENV=production node server.js"
   },
+  "jest": {
```
Comment (anshurathee) [inline package.json]: Make a separate jest.config.js file in the root of the repo. Code all the configurations there.
http://jestjs.io/docs/en/configuration

**Review by anshurathee — CHANGES_REQUESTED** [review]

File: package.json [hunk 3]
```diff
@@ -6,29 +6,52 @@
...
+    "setupFiles": [
+      "./shim.js",
+      "./test-setup.js"
+    ]
+  },
+  "lint-staged": {
```
Comment (anshurathee) [inline package.json]: Similarly move the lint-staged config to a .lintstagedrc file

**Review by anshurathee — APPROVED** [review]

**Review by ArfatSalman — APPROVED** [review]

**Pull request merged**